contracts/ReputationMiningCycle.sol

```diff
@@ -83,7 +83,7 @@ contract ReputationMiningCycle is ReputationMiningCycleStorage, PatriciaTreeProo
   /// At the beginning of the submission window, the target is set to 0 and slowly increases to 2^256 - 1 after an hour
   modifier withinTarget(bytes32 newHash, uint256 entryIndex) {
     // Check the ticket is a winning one.
-    // All entries are acceptable if the hour-long window is closed, so skip this check if that's the case
+    // All entries are acceptable if the 24 hour-long window is closed, so skip this check if that's the case
```

Comment:
IMO let's not even mention the window length in the comments, it's just another place for things to get out-of-date.